packages/bot-web-ui/src/xml/dalembert_max-stake.xml

```diff
@@ -435,8 +434,8 @@
                         </value>
                         <statement name="DO0">
                           <block type="variables_set" id="iX^w=Jc(b/VlAIdNMduh">
-                            <field name="VAR" id="o((rBg*rL/qm;RU%PM7_" variabletype="">dalembert:alembert_unit</field>
-                            <value name="VALUE">
+                            <field name="VAR" id="o((rBg*rL/qm;RU%PM7_" variabletype="">dalembert:unit</field>
+                            <value name="VALUE" strategy_value="dalembert_unit">
```

Comment:
I noticed that one field was missing, filling in a new value from the quick strategy form did not happen.
Fixed additionally✅